samples/expert-othello-adjusted.rb

```diff
@@ -262,14 +263,13 @@ def draw_board
             stroke rgb(0, 100, 0)
             rect left: left, top: top, width: PIECE_WIDTH, height: PIECE_HEIGHT
 
-            if cell != 0
-              strokewidth 0
-              fill(cell == 1 ? rgb(100,100,100) : rgb(155,155,155))
-              oval(left+3, top+4, PIECE_WIDTH-10, PIECE_HEIGHT-10)
+            next unless cell != 0
```

Comment:
Yeah, for this sample in particular I feel like the `next` isn't nearly as obvious. I'd be onboard with disabling that particular rule.